Mathlib/LinearAlgebra/FiniteDimensional.lean

```diff
@@ -64,7 +64,7 @@ in `Mathlib.LinearAlgebra.Dimension`. Not all results have been ported yet.
 You should not assume that there has been any effort to state lemmas as generally as possible.
 
 Plenty of the results hold for general fg modules or notherian modules, and they can be found in
-`Mathlib.LinearAlgebra.FreeModule.Finite.Rank` and `Mathlib.RingTheory.Noetherian`.
+`Mathlib/LinearAlgebra/FreeModule/Finite/Rank.lean` and `Mathlib/RingTheory/Noetherian.lean`.
```

Comment:
I think this should be reverted
```suggestion
`Mathlib.LinearAlgebra.FreeModule.Finite.Rank` and `Mathlib.RingTheory.Noetherian`.
```
We generally refer to files by their `lake` module name.

Reply:
We're not consistent at all. But apparently the reason this doesn't link to the module in the generated docs is that `Mathlib.LinearAlgebra.FreeModule.Finite.Rank` doesn't even exist anymore, rather than the syntax not being supported. I'll revert this and fix the links elsewhere.